src/sidebar/components/tag-editor.js

```diff
@@ -45,12 +52,23 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
     return suggestionsSet.sort();
   };
 
-  // List of suggestions returned from the tagsService
-  const suggestions = useMemo(() => {
-    // Remove any repeated suggestions that are already tags.
-    // Call filter with an empty string to return all suggestions
-    return removeDuplicates(tagsService.filter(''), tagList);
-  }, [tagsService, tagList]);
+  /**
+   * Get a list of suggestions returned from the tagsService
+   * reset the activeItem and open the Datalist
+   */
+  const updateSuggestions = () => {
+    const value = inputEl.current.value.trim();
+    if (value === '') {
+      // If there is no input, just hide the suggestions
+      setSuggestionsListOpen(false);
+    } else {
+      // Remove any repeated suggestions that are already tags.
+      // Call filter with an empty string to return all suggestions
```

Comment:
The "Call filter" comment refers to just a small part of the next line, which slightly confused me. It would help to separate out the `tagsService.filter` call onto a separate line which the comment can refer to on its own.